frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -173,19 +177,20 @@ export default function CreateBooking() {
       ) : (
         <Form onSubmit={onSubmit}>
           <Controller
```

Comment:
point was to update input field so we would not need to use controller to validate the input

Reply:
So you're saying you want to use ref=register() instead of Controller?
But what is the difference we can achieve by doing so?

https://stackoverflow.com/questions/63264426/what-are-major-differences-between-controlled-and-uncontrolled-components-in-rea

Reply:
i think it's more cumbersome to use controller since there are so many things to keep track of instead of using ref directly; also there is likely more room for errors this way

Reply:
![image](https://user-images.githubusercontent.com/43675202/158632566-304685a3-f943-4e2f-a779-1ae70bef704f.png)
Okay I changed it to register. Is this the correct way to do it?

Reply:
ok i realized if we use register then we will have to do setValue, setError and clearError manually which defeats the point cos i think Antd Components dont work well with register so i would think that using Controller is a better way